### DISCUSSION TITLE
Is multi-language support planned for F5-TTS?

### DISCUSSION THREAD

**Discussion by Entity303GB** [SWivid/F5-TTS#217] [Q&A]
Hello, I am really impressed with F5-TTS and was wondering if there are any plans to support multiple languages in the future. Is this on the roadmap, or should we not expect this feature anytime soon? Thanks!

**Comment by SWivid** [comment]
planned √

**Reply by Entity303GB** [reply]
**Hi again! Thanks for the quick reply!**
I'm glad to hear that multiple languages are planned! Do you have any rough estimate on when this feature might be released? Also, is there a Discord server or any other platform where you share updates and news about the project or is it only on GitHub?
Thanks again!

**Reply by leoiania** [reply]
@SWivid is there a way in which the community could actively help/contribute to this? I would like to support!

**Reply by kunibald413** [reply]
first japanese finetune by https://github.com/JarodMica: https://github.com/SWivid/F5-TTS/discussions/57#discussioncomment-11050162
